AnnoDesigner/model/LayoutObject.cs

```diff
@@ -0,0 +1,391 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.model
+{
+    public class LayoutObject
+    {
+        private AnnoObject _wrappedAnnoObject;
+        private readonly ICoordinateHelper _coordinateHelper;
+
+        private Color? _transparentColor;
+        private SolidColorBrush _transparentBrush;
+        private Color? _renderColor;
+        private SolidColorBrush _renderBrush;
+        private Pen _borderlessPen;
+        private int _gridSizeScreenRect;
+        private Point _position;
+        private Rect _screenRect;
+        private Rect _collisionRect;
+        private Size _collisionSize;
+        private string _iconNameWithoutExtension;
+        private string _identifier;
+        private Size _size;
+        private FormattedText _formattedText;
+        private CultureInfo _usedTextCulture;
+        private Typeface _usedTextTypeFace;
+        private Rect _iconRect;
+        private int _gridSizeIconRect;
+        private Rect _lastScreenRectForIcon;
+        private Point _screenRectCenterPoint;
+        private EllipseGeometry _influenceCircle;
+        private double _influenceCircleRadius;
+        private Rect _lastScreenRectForCenterPoint;
+        private double _screenRadius;
+        private int _lastGridSizeForScreenRadius;
+        private SerializableColor _color;
+
+        public LayoutObject(AnnoObject annoObjectToWrap, ICoordinateHelper coordinateHelperToUse)
```

Comment:
Add comments to this making sure to note the behaviour of the wrapper - that it keeps a reference to original AnnoObject passed in to the constructor.

Reply:
Good catch.